src/Cohesity.Powershell/Cohesity.PowerShell.psd1

```diff
@@ -132,11 +132,12 @@ FunctionsToExport = @(
                     'Remove-CohesityVirtualIP',
                     'Remove-CohesityVlan',
                     'Restore-CohesityBackupToView',
-                    'Restore-CohesityFileV2',                   
+                    'Restore-CohesityFileV2',
+                    'Restore-CohesityOracleDatabase',
                     'Restore-CohesityRemoteFile',
                     'Restore-CohesityRemoteFileV2',
                     'Restore-CohesityRemoteMSSQLObject',
-                    'Restore-CohesityOracleDatabase',
```

Comment:
This breaks compatibility?

Reply:
No. Just arranged the commandlet ahabetically.

Reply:
My bad. didnt see it correctly.